atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
```

Comment:
So I'd initially dropped out the 'if fs_result is None' bit, and things went bad with self-tests. If there's no task_id, we should already return None before we go through the whole exception dance. If something is returned, and it's not a valid task ID, I think we do want to explode, because that shouldn't happen, right?